sdk/ai/azure-ai-generative/azure/ai/generative/evaluate/_utils.py

```diff
@@ -26,6 +30,48 @@ def load_jsonl_to_df(path):
     else:
         raise Exception("File not found: {}".format(path))
 
+def df_to_dict_list(df, extra_kwargs: Dict = None):
+    if extra_kwargs is not None:
+        return df.assign(**extra_kwargs).to_dict("records")
+    return df.to_dict("records")
+
+
+def run_pf_flow_with_dict_list(flow_path, data: List[Dict], flow_params=None):
```

Comment:
is `flow_path` pointing to a local PF or to a registered built-in flow?

Reply:
It needs to be a local flow for now.